run_servers.sh

```diff
@@ -10,11 +10,22 @@ require() {
   fi
 }
 
-require bindle-server
-require consul
-require nomad
-require traefik
+require_root() {
+  if ! sudo bash -c "hash $1" &>/dev/null; then
+    echo "'$1' not found in root user's PATH"
+    exit 1
+  fi
+}
+
+# The following will be run (either directly or indirectly) via sudo, and thus must appear in the root user's PATH:
+require_root bindle-server
+require_root nomad
+require_root traefik
+require_root spin
+
+# The following will be run directly as the current user, and thus must appear in the current PATH:
 require vault
+require consul
```

Comment:
slight nitpick here, but can we put this above line 27 so the `require` lines are all sorted alphabetically?